client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx

```diff
@@ -8,24 +8,23 @@ import styles from './BatchChangesListIntro.module.scss'
 export const BatchChangesChangelogAlert: React.FunctionComponent = () => (
     <DismissibleAlert
         className={styles.batchChangesListIntroAlert}
-        partialStorageKey="batch-changes-list-intro-changelog-3.29"
+        partialStorageKey="batch-changes-list-intro-changelog-3.30"
     >
         <div className={classNames(styles.batchChangesListIntroCard, 'card h-100 p-2')}>
             <div className="card-body">
-                <h4>New Batch Changes features in version 3.29</h4>
+                <h4>New Batch Changes features in version 3.30</h4>
                 <ul className="mb-0 pl-3">
-                    <li>New bulk operations have been added to retry or merge multiple changesets at once. &#x2705;</li>
                     <li>
-                        <a target="_blank" rel="noopener noreferrer" href="https://github.com/sourcegraph/src-cli">
-                            <code>src</code>
-                        </a>{' '}
-                        can now cache the result of each step when executing a batch spec, rather than only caching the
-                        final result. &#x1f680;
-                    </li>
-                    <li>
-                        Changeset specs on batch changes that have been previewed but not applied now expire
-                        consistently after one week. &#x1f5d3;&#xfe0f;
+                        Changesets can now be published from the UI!{' '}
+                        <a
+                            href="https://docs.sourcegraph.com/batch_changes/how-tos/publishing_changesets#within-the-ui"
+                            rel="noopener"
+                            target="_blank"
+                        >
+                            Learn more.
+                        </a>
                     </li>
+                    <li>You can now view open changesets per repository from the repository landing page.</li>
```

Comment:
Thanks for the heads up Erik!! Is our cutoff tomorrow at 6PM UTC when the calendar event is, or do we want to lock it in earlier? It's going to be really close haha...

```suggestion
                    <li>You can now view changesets created in a specific repository from that repository's landing page.</li>
```

Reply:
I hope so, the past has proven it sometimes happens a bit earlier, so I wouldn't jinx it 😬 
Let me know if I can help or whether I should drop it from here!

Reply:
Okay thanks! I'll let you know by EOD!